discord_bots/cogs/admin.py

```diff
@@ -0,0 +1,692 @@
+            )
+            if exists is None:
+                await interaction.response.send_message(
+                    embed=Embed(
+                        description="Could not find a command with that name",
+                        colour=Colour.red(),
+                    ),
+                    ephemeral=True,
+                )
+                return
+
+            exists.output = output
+            session.commit()
+
+        await interaction.response.send_message(
+            embed=Embed(
+                description=f"Command `{name}` updated",
+                colour=Colour.green(),
+            )
+        )
+
+    @group.command(
+        name="editgamewinner", description="Edit the winner of a finished game"
+    )
+    @app_commands.check(is_admin_app_command)
+    @app_commands.check(is_command_channel)
+    @app_commands.describe(game_id="Finished game id", outcome="<tie|be|ds>")
+    async def editgamewinner(
+        self,
+        interaction: Interaction,
+        game_id: str,
+        outcome: Literal["Tie", "BE", "DS"],
+    ):
+        session: SQLAlchemySession
+        with Session() as session:
+            game: FinishedGame | None = (
+                session.query(FinishedGame)
+                .filter(FinishedGame.game_id.startswith(game_id))
+                .first()
+            )
+            if not game:
+                await interaction.response.send_message(
+                    embed=Embed(
+                        description=f"Could not find game: {game_id}",
+                        colour=Colour.red(),
+                    ),
+                    ephemeral=True,
+                )
+                return
+            outcome_lower = outcome.lower()
+            if outcome_lower == "tie":
+                game.winning_team = -1
+            elif outcome_lower == "be":
+                game.winning_team = 0
+            elif outcome_lower == "ds":
+                game.winning_team = 1
+            else:
+                await interaction.response.send_message(
+                    embed=Embed(
+                        description="Outcome must be tie, be, or ds",
+                        colour=Colour.red(),
+                    ),
+                    ephemeral=True,
+                )
+                return
+
+            session.add(game)
+            session.commit()
+        await interaction.response.send_message(
+            embed=Embed(
+                description=f"Game {game_id} outcome changed:\n\n"
+                + finished_game_str(game),
+                colour=Colour.green(),
+            )
+        )
+
+    @group.command(name="remove", description="Remove an admin")
+    @app_commands.check(is_admin_app_command)
+    @app_commands.check(is_command_channel)
+    @app_commands.describe(member="Player to be removed as admin")
+    async def removeadmin(self, interaction: Interaction, member: Member):
```

Comment:
For the grouping, there was several ways it could've been done, but I decided on just having it be alphabetical in most circumstances. 
I think `utils.py` might be the only exception to that.

Further subdivision / grouping can certainly be looked at down the road, but as it is, none of these cogs exceed the 25 command maximum.

Reply:
As we've discussed over discord, further command grouping can come down with more general refactors to the _logic_ of the commands. I don't really care about the ordering within the files, since the darker doesn't do it for you and you can always use an IDE or grep to find whatever one you want. Of course that would be nice to have, but unless there's some automated way to do it via the pre-commit hook, then I don't want anyone to have to do that themselves by hand. Alphabetical is preferred